packages/cli/src/ui/components/InputPrompt.test.tsx

```diff
@@ -407,4 +409,26 @@ describe('InputPrompt', () => {
     expect(props.onSubmit).not.toHaveBeenCalled();
     unmount();
   });
+
+  it('should call buffer.undo on Ctrl+G', async () => {
```

Comment:
make these tests a little more robust by adding some text to undo and redo and verifying that the text is actually undone and redone.